js/views/cfi_navigation_logic.js

```diff
@@ -1214,7 +1214,7 @@ var CfiNavigationLogic = function (options) {
                             if (!firstPartial) firstPartial = item;
 
                             if (visible == 100) return item;
-                        }
+                        } else return item;
```

Comment:
What is the reason for this change? This seems to run counter to finding the first *visible* Media Overlays element. Without this code change, this is equivalent to `continue` (with respect to the enclosing loop), but with the change the first non-visible MO element will be found!